src/routes/sessions/[id]/Prompt.svelte

```diff
@@ -1,27 +1,45 @@
 <script lang="ts">
```

Comment:
There are several type errors on this file.

Reply:
About this, I don't really know what the problem with the linter is. It seems like the `Knowledge` component is not being imported correctly.

However, the syntax looks fine (the component is imported from `Controls` as well and there isn't any problem there), the browser does not throw any errors, and if I run `npm run lint` it says that all files are okay.

I don't know how to get rid of that annoying lint error in the code editor.